CHANGELOG.md

```diff
@@ -1,9 +1,10 @@
-# 2.6.1 (not released)
+# 2.6.1 (2020-09-08)
```

Comment:
no: move all this section in 2.6.0 and change package version to be 2.6.0 so it can be released.